'

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.feature "Visitor can sign up for a account" do
+  scenario "they visit signup page and create a account and sees user dashboard" do
+    visit login_path
+
+    fill_in "fullname", with: "Nate Venn"
+    find("input[placeholder='email']").set "nate@example.com"
+    find("input[placeholder='password']").set "password"
```

Comment:
@Salvi6God this looks like a new test he wrote. i talked to @natevenn and we are gonna merge